browser/ui/webui/playlist_ui.cc

```diff
@@ -123,7 +126,9 @@ bool PlaylistUI::ShouldBlockPlaylistWebUI(
     return false;
   }
 
-  return !PlaylistServiceFactory::IsPlaylistEnabled(browser_context);
+  return !PlaylistServiceFactory::IsPlaylistEnabled(browser_context) ||
```

Comment:
you should be checking to see if the service is null from the factory and `IsPlaylistEnabled` should be removed